EventKit/eventkit-cloud
# Allow data providers to be filtered by topic when creating datapack

## Description of Improvement

Adds initial representation of topic model for frontend, and enables filtering providers by topic when creating a datapack.

## How to test

Add some test topics linked to providers
Go to CreateDatapack page
verify topics appear in filter section
verify providers filter when topic is selected
Ensure all tests pass

## Quality Assurance 

The following items have been updated:
 
## Screenshots

:smile:
